landing-pages/site/layouts/meetups/list.html

```diff
@@ -1,79 +1,75 @@
 {{/*
- Licensed to the Apache Software Foundation (ASF) under one
- or more contributor license agreements.  See the NOTICE file
- distributed with this work for additional information
- regarding copyright ownership.  The ASF licenses this file
- to you under the Apache License, Version 2.0 (the
- "License"); you may not use this file except in compliance
- with the License.  You may obtain a copy of the License at
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
 
-   http://www.apache.org/licenses/LICENSE-2.0
+      http://www.apache.org/licenses/LICENSE-2.0
 
- Unless required by applicable law or agreed to in writing,
- software distributed under the License is distributed on an
- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- KIND, either express or implied.  See the License for the
- specific language governing permissions and limitations
- under the License.
-*/}}
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+   */}}
 
-{{ define "main" }}
-    <h2 class="page-header">Meetups</h2>
-    <h5 class="meetups-page page-subtitle">
-        Meetups are a great way for the community to meet face-to-face
-    </h5>
-    <div class="meetups">
-        <script type="application/x-template" id="meetup-template">
-            <div class="list-item">
-                <div class="card">
-                    <div class="box-event">
-                        <p class="box-event__meetup--location" data-name="location"></p>
-                        <p class="box-event__meetup--members">
+   {{ define "main" }}
```

Comment:
Could you check your linter? I don't think the indentation should've changed.

Reply:
Thanks for catching this and the typos, @bbovenzi . This is RFR again.